src/misc/falling.ts

```diff
@@ -0,0 +1,11 @@
+import { LivingEntity } from 'org.bukkit.entity';
+import { EntityDamageEvent } from 'org.bukkit.event.entity';
+import { PotionEffect, PotionEffectType } from 'org.bukkit.potion';
+
+const slow = new PotionEffect(PotionEffectType.SLOW, 4 * 20, 2);
```

Comment:
```suggestion
const SLOW_EFFECT = new PotionEffect(PotionEffectType.SLOW, 4 * 20, 2);
```
Capitalize constant names.